1.0/docs/start/quick-tour.md

```diff
@@ -320,11 +320,13 @@ to the element's `owner` property, so it's updated as the user types.
   </div>
 </demo-tabs>
 
-**Note:** The `is="iron-input"` attribute indicates the input is a _type-extension_ custom 
-element; the element name is `iron-input`, and it _extends_ the native `<input>` element. 
+**Note:** The `is="iron-input"` attribute indicates the input is a 
+_type-extension_ custom element; the element name is `iron-input`, and it 
+_extends_ the native `<input>` element. To get this sample working you would 
+need to [install `iron-input` with Bower](getting-the-code.html#project-setup) 
```

Comment:
this page doesn't describe how to install iron-input, so the link text is off. Either link just "Bower" (since this is more-or-less the definitive reference to bower on the site), Or move the link to a new sentence at the of the paragraph, saying, "For information on Bower, see [Get the code](...)"